docs/resources/flights.md

```diff
@@ -699,7 +699,9 @@ The response is pretty straightforward:
 
 With the [Flight Availabilities Search API](https://developers.amadeus.com/self-service/category/air/api-doc/flight-availabilities-search) you can check the flight and fare availability for any itinerary. This refers to the full inventory of fares available for an itinerary at any given time. The concept of flight availability originated in the early days of flight booking as a way for agents to check what options existed for their travelers’ itineraries.
 
-You can build the request by passing the flight-offer object from the [Flight Offers Search API](https://developers.amadeus.com/self-service/category/air/api-doc/flight-offers-search) into the body of the `POST` request. Here’s an example request for a one-way flight from Mad (MIA) to Atlanta (ATL) for one traveler departing on December 12, 2021: 
+You can build the request by passing into the body of the `POST` request a flight-offer object that you can customise to your needs. An example of such object is provided in the [Flight Offers Search API](https://developers.amadeus.com/self-service/category/air/api-doc/flight-offers-search). In addition to this, a `X-HTTP-Method-Override` header parameter is required.
```

Comment:
Hey Nico, is not really a flight-offer object that we pass. Check the img from the swagger. Is just that the POST Flight Offers Search and POST Flight Availabilities Search have almost the same input ![Screenshot 2023-01-26 at 12 10 22](https://user-images.githubusercontent.com/15012967/214821835-f754752d-88a5-4e49-9dc3-3e3b67ee85bf.png)